MagazineLayout/Public/MagazineLayout.swift

```diff
@@ -265,32 +261,37 @@ public final class MagazineLayout: UICollectionViewLayout {
     supplementaryViewLayoutAttributesForPendingAnimations.removeAll()
 
     targetContentOffsetAnchor = nil
+    preInvalidationContentSize = nil
+
+    if let stagedContentOffsetAdjustment {
+      let context = MagazineLayoutInvalidationContext()
+      context.invalidateLayoutMetrics = false
+      context.contentOffsetAdjustment = stagedContentOffsetAdjustment
+      invalidateLayout(with: context)
+    }
+    stagedContentOffsetAdjustment = nil
```

Comment:
This was a key insight from https://itnext.io/my-covid-19-lockdown-project-or-how-i-started-to-dig-into-a-custom-uicollectionviewlayout-to-get-a-d053e1ad3aa0 (there's a comment describing it below)

Reply:
I'm not sure I'm seeing it, maybe we should add this in a comment?